nixos/modules/services/web-apps/hledger-web.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, pkgs, config, ... }:
+with lib;                      
+let
+  cfg = config.services.hledger-web;
+in {
+  options.services.hledger-web = {
+
+    enable = mkEnableOption "hledger-web service";
+
+    serveApi = mkEnableOption "Serve only the JSON web API, without the web UI.";
+
+    host = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = ''
+        Address to listen on.
+      '';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 5000;
+      example = "80";
+      description = ''
+        Port to listen on.
+      '';
+    };
+
+    capabilities = mkOption {
+      type = types.commas;
+      default = "view,add";
```

Comment:
I don't think that capabilities that change the journal, like `add`, should be enabled by default.

Reply:
removed the add capability